Removed parsing of vendor folder, by default. Added supports of './' in `except` & `only` (#16)
yiisoft/translator-extractor: README.md

php yii translator/extract /path/to/your/project
```

> **Notice:** By default we exclude folder `vendor` in root folder. For remove this behaviour you can specify empty value of `except`

Reviewer: ```suggestion
**Notice:** By default extractor has `vendor` directory in the application directory excludede. To include it you can specify empty value for `except`:
```

### Using `except` option

> To exclude all folders with name `dir1` directory use `--except`:

Reviewer: ```suggestion
To exclude all directories named `dir1` use `--except`:
```
